src/vivarium_public_health/population/data_transformations.py

```diff
@@ -54,23 +55,23 @@ def assign_demographic_proportions(
 
     population_data["P(sex, location, age| year)"] = (
         population_data.groupby("year_start", as_index=False)
-        .apply(lambda sub_pop: sub_pop.value / sub_pop.value.sum())
-        .reset_index(level=0)
-        .value.fillna(0.0)
+        .apply(lambda sub_pop: sub_pop[["value"]] / sub_pop["value"].sum())
+        .reset_index(level=0)["value"]
```

Comment:
I updated the value column dot-notation (`.value`) with bracket notation (`["value"]`) for consistency w/ how we usually do things.